packages/react-native-bridge/android/react-native-bridge/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="org.wordpress.mobile.ReactNativeGutenbergBridge">
```

Comment:
The React Native CLI fetches the `package` attribute from AndroidManifest files, so we can't remove it.

This can be tested by running `npx react-native config`.

Reply:
> I think it is okay (safe) to keep this package definition here @fluiddot , that is, as long as we make sure to have the same namespace defined on the inner `react-native-bridge/build.gradle` level, just we already did. 👍
> 
> PS: We just need to not forget to update one without the other, that is, if we ever need to do something like that anyway (but I doubt that this will be happening anytime soon). 🤷

Good point. I think having an inline comment in both `build.gradle` and `AndroidManifest.xml` files reminding this would be helpful to avoid forgetting that both need to be updated. I'll add this 💨 .

Reply:
I applied my above suggestion in https://github.com/WordPress/gutenberg/pull/52872/commits/67ca6bdc8251773153889ac9711f8adfba76a93b.